datafusion/common/src/config.rs

```diff
@@ -271,7 +271,7 @@ config_namespace! {
         /// for each output file being worked. Higher values can potentially
         /// give faster write performance at the cost of higher peak
         /// memory consumption
-        pub max_buffered_batches_per_output_file: usize, default = 2
+        pub max_buffered_batches_per_output_file: usize, default = 10
```

Comment:
This seems very high

Reply:
I lowered this one back to 2 with no noticeable impact on performance.